docs/run/cas/community-cas.md

```diff
@@ -0,0 +1,54 @@
+# Community CAS
+This page contains a list of free [Ceramic Anchor Services (CAS)](../../learn/glossary.md#anchor-service) hosted by [3Box Labs](https://3boxlabs.com) and made available for the community. The CAS referenced below are included by default in every [Ceramic node](../../learn/glossary.md#nodes) and will be automatically used depending on the node's network configuration; no additional setup is needed. At this time it is not possible to run your own CAS, however this functionality will be available in the near future.
+
+## **Mainnet CAS**
+Nodes connected to [Mainnet](../../learn/networks.md#mainnet) will automatically use this CAS which generates [anchor commits](../../learn/glossary.md#anchor-commit) using the Ethereum Mainnet blockchain (`eip155:1`) and communicates over the `/ceramic/mainnet` libp2p topic. 
+
+!!! ""
+
+    **Mainnet usage is currently restricted to projects in the Early Launch program (ELP).** Sign up for the ELP waitlist [here](https://blog.ceramic.network/ceramic-mainnet-early-launch-program/).
+    
+
+HTTP Endpoint:
+
+```
+https://cas-internal.3boxlabs.com
```

Comment:
I don't think we should include the http endpoints here, because we don't want people making requests to them outside of using a Ceramic node which already has them hardcoded.

But also the mainnet CAS is not accessible so when people hit this they will get an error so we should definitely not include it.

Reply:
Okay I removed all endpoints